client/src/components/Header/index.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import { IconButton, makeStyles } from "@material-ui/core";
+import PersonIcon from "@material-ui/icons/Person";
+import styles from "./style.module.scss";
+import FocusLogo from "../../images/focus-logo.svg";
+import SearchBar from "../SearchBar";
```

Comment:
Would it be cleaner to nest the SearchBar component folder inside the Header folder or do you think that is unnecessary?